src/AuctionHouseBot.cpp

```diff
@@ -26,6 +26,7 @@
 #include <vector>
 
 using namespace std;
+vector<SimpleItemConfigEntry> simpleItemConfig;
```

Comment:
I would recommend turning on 'hide whitespace changes' when reviewing this file as I added a few blocks of code like if statements and loops that required changing some indentation.